ci/generate_docs.sc

```diff
@@ -38,6 +41,9 @@ def main(release_commits_override: Map[String, String] = Map.empty) = {
     println(release_commits_override.map(e => s"Version ${e._1}: ${e._2}").mkString("\n"))
   }
 
+  if (publish) {
+    println(s"Documentation will be published to $remote")
```

Comment:
Shoudln't this be part of `def publish`?

Reply:
Push. I'd move this line to the method.

Reply:
We already log it in publish method, the point of this message is to warn the user so he/she has time to cancel the build.